File: MyHeart.Web/ClientApp/models/professionInfo/SymptomQuestionsDto.ts
import SymptomQuestionsSymptomDto from "./bonds/SymptomQuestionsSymptomDto";
import SymptomQuestionsDiseaseDto from "./bonds/SymptomQuestionsDiseaseDto";

export default class SymptomQuestionsDto {
  id: number;
    text: string | null;
    symptoms: SymptomQuestionsSymptomDto[];
    diseases: SymptomQuestionsDiseaseDto[];
    symptomName: string | null;
    diseaseString: string | null;
}

File: MyHeart.DTO/ProfessionInformation/SymptomQuestionsDTO.cs
using MyHeart.DTO.ProfessionInformation;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace MyHeart.DTO.ProfessionalInformation
{
    public class SymptomQuestionsDTO : BaseEntityDTO
    {
        public string Text { get; set; }
        public List<SymptomQuestions_SymptomDTO> Symptoms { get; set; }
        public List<SymptomQuestions_DiseaseDTO> Diseases { get; set; }
        public string SymptomName { get; set; }
        public string DiseaseString { get; set; }
    }
}
